src/main/java/com/google/summit/ast/SourceLocation.kt

```diff
@@ -21,6 +21,11 @@ package com.google.summit.ast
  *
  * A special "unknown" value can also be constructed.
  *
+ * The range is inclusive of the end character.
+ * The line numbering begins at 1.
+ * The column numbering begins at 0.
```

Comment:
I never thought carefully about this: is this lack of symmetry "standard"? And/or are you going with this choice for compatibility with some other layer (antlr, pmd)?

Reply:
Not sure, but ANTLR and PMD disagree. Summit copied the behavior of ANTLR.

https://www.antlr.org/api/Java/org/antlr/v4/runtime/Token.html#getCharPositionInLine()